pyodide-build/pyodide_build/mkpkg.py

```diff
@@ -89,23 +89,12 @@ def _get_metadata(package: str, version: str | None = None) -> dict:
     return pypi_metadata
 
 
-def _import_ruamel_yaml():
-    """Import ruamel.yaml with a better error message is not installed."""
-    try:
-        from ruamel.yaml import YAML
-    except ImportError as err:
-        raise ImportError(
-            "No module named 'ruamel'. "
-            "It can be installed with pip install ruamel.yaml"
-        ) from err
-    return YAML
-
-
 def run_prettier(meta_path):
     subprocess.run(["npx", "prettier", "-w", meta_path])
 
 
 def make_package(
+    root: Path,
```

Comment:
Maybe better to pass in `packages_dir` here.

Reply:
Yeah, that sounds more appropriate. Thanks!